### PR TITLE
fix click favorite no response when log out

### PR DESCRIPTION
**Pull request by uwgnol1612**

### CODE REVIEW

File: client/src/components/main/Favorite/AddShopToFavorite.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ const AddShopToFavorite = props => {
   return (
     <Query query={FETCH_CURRENT_USER}>
       {({ data, loading, error }) => {
-        if (loading) return "Loading...";
+        if (loading) return 'Loading...';
+        debugger;
```
Comment (jmsteen) [inline client/src/components/main/Favorite/AddShopToFavorite.js]: please remove debugger

File: client/src/components/main/Favorite/AddShopToFavorite.js [hunk 2]
```diff
@@ -42,24 +43,29 @@ const AddShopToFavorite = props => {
...
-                  />
+                  <AuthLink
+                    to={`/shop/${props.shopId}`}
+                    content={
+                      <img
+                        src="https://we-camp-seeds.s3.us-east-2.amazonaws.com/favorite.png"
+                        onClick={() => {
+                          removeFavorite({
+                            variables: {
+                              // userId: currentUserId,
```
Comment (jmsteen) [inline client/src/components/main/Favorite/AddShopToFavorite.js]: Do you want to keep this comment?

**Review by jmsteen — CHANGES_REQUESTED** [review]
couple of minor comments

File: client/src/components/main/Favorite/AddShopToFavorite.js (hunk 1 above)
Comment (uwgnol1612) [reply client/src/components/main/Favorite/AddShopToFavorite.js]: sorry, I remove it in the new pr

**Review by jmsteen — APPROVED** [review]

**Pull request merged**